.env

```diff
@@ -1,3 +1,4 @@
+# Thank you for your concern, these api keys are intentionally public. Please do not report them.
```

Comment:
Put the statement of fact in front:

    These API keys are intentionally public. Please do not report them - thank you for your concern.